app-editors/vis/vis-0.1.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2017 Gentoo Foundation
```

Comment:
Is there any specific reason you are adding three versions? If you do not have a very good reason to do this, just add the newest. Gentoo users will not install the older versions anyway.